tests/discover/filtering.sh

```diff
@@ -190,24 +190,6 @@ in plan \`/plans/a-non-url\` can be used only within git repo." output
         rlRun "tmt run -rdvvv discover -h fmf --fmf-id \
                --url https://github.com/teemtee/fmf finish 2>&1 >/dev/null | tee output" 0
         rlRun "rm -rf $tmp_dir1 $tmp_dir2"
-
-        # 2: w/ url in plan AND w/o url in plan: w/o url in CLI - w/ url in CLI
-        tmp_dir1="$(mktemp -d)"
-        tmp_dir2="$(mktemp -d)"
-        rlRun "cd $tmp_dir1"
-        rlRun "tmt init --template full"
-        rlRun "cd $tmp_dir2"
-        rlRun "tmt init --template base"
-        rlRun "cp plans/example.fmf $tmp_dir1/plans/z-non-url.fmf"
-        rlRun "cd $tmp_dir1"
-        rlRun "tmt run -rdvvv discover -h fmf --fmf-id finish 2>&1 >/dev/null \
-               | tee output" 2
-        rlAssertGrep "\`tmt run discover --fmf-id\` without \`url\` option \
-in plan \`/plans/z-non-url\` can be used only within git repo." output
-
-        rlRun "tmt run -rdvvv discover -h fmf --fmf-id \
-               --url https://github.com/teemtee/fmf finish 2>&1 >/dev/null | tee output" 0
-        rlRun "rm -rf $tmp_dir1 $tmp_dir2"
```

Comment:
@bajertom why are we dropping this test?

Reply:
Even if the code change, maybe we should cover the case when `url` is not specified?

Reply:
@thrix, see: https://github.com/teemtee/tmt/pull/3701#discussion_r2172133003